components/list/test/list-item-drag-image.visual-diff.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script type="module">
+			import '../list-item-drag-image.js';
+		</script>
+		<title>d2l-list-item-drag-image</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<script type="module">
+			import '../../typography/typography.js';
+		</script>
+		<style>
+			.visual-diff {
+				height: 80px;
+				position: relative;
+				width: 350px;
+			}
+			d2l-list-item-drag-image {
+				left: 0 !important;
```

Comment:
This is simply to override the internal offscreen positioning so that we can take a screen shot.